sources/packages/backend/apps/api/src/route-controllers/program-year/_tests_/e2e/program-year.students.controller.getProgramYears.e2e-spec.ts

```diff
@@ -0,0 +1,157 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import { ProgramYear } from "@sims/sims-db";
+import * as request from "supertest";
+import { DataSource } from "typeorm";
+
+import {
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getStudentToken,
+  FakeStudentUsersTypes,
+} from "../../../../testHelpers";
+
+describe("ProgramYearStudentsController(e2e)-getProgramYears", () => {
+  let app: INestApplication;
+  let appDataSource: DataSource;
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    appDataSource = dataSource;
+    app = nestApplication;
+  });
+
+  it("Should get the list of the available program years.", async () => {
+    // Arrange
+    const endpoint = "/students/program-year/options-list";
+    const studentToken = await getStudentToken(
+      FakeStudentUsersTypes.FakeStudentUserType1,
+    );
+
+    // Act/Assert
+    await request(app.getHttpServer())
+      .get(endpoint)
+      .auth(studentToken, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.OK)
+      .then((response) => {
+        const programYearList = response.body;
+        expect(programYearList).toHaveLength(3);
+        expect(programYearList).toEqual(
```

Comment:
I would recommend doing the check following the `expected` sntax as below. IMO it is simpler and cleaner. It will eliminate the need to check the 36 also.
![image](https://user-images.githubusercontent.com/61259237/225446324-30c6feb3-6c13-4d16-9f0e-827fa25b5b3e.png)